sdk/keyvault/Azure.Security.KeyVault.Administration/src/Generated/AzureSecurityKeyVaultAdministrationModelFactory.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+
+namespace Azure.Security.KeyVault.Administration
+{
+    /// <summary> Model factory for AzureSecurityKeyVaultAdministration read-only models. </summary>
+    public static partial class AzureSecurityKeyVaultAdministrationModelFactory
```

Comment:
This is the wrong name. There's already another PR where it's being addressed. Can we rebase on master as soon as that's merged?